tcontainer/arrays.go

```diff
@@ -62,9 +60,7 @@ func (s Uint64Slice) Less(i, j int) bool {
 }
 
 func (s Uint64Slice) Swap(i, j int) {
-	s[i] = s[i] ^ s[j]
-	s[j] = s[j] ^ s[i]
-	s[i] = s[i] ^ s[j]
+	s[i], s[j] = s[j], s[i]
 }
```

Comment:
There is a lot more assembly generated by the compiler for the xor version so although I could not measure any difference more instructions = not good.